kft-2020/README.md

```diff
@@ -0,0 +1,18 @@
+# Kubernetes Fest Tokyo 2020: Kubernetes Upstream Training
+
+<a href="https://github.com/kubernetes/kubernetes"><img src="https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png" width="64"></a><a href="xxx"><img src="./images/xxx.png"></a>
```

Comment:
It's better to use the Markdown format : https://guides.github.com/features/mastering-markdown/#examples

Reply:
Thank you to comment.
I've fixed it.